src/storage/txn/actions/flashback_to_version.rs

```diff
@@ -225,6 +232,25 @@ pub fn commit_flashback_key(
     Ok(())
 }
 
+pub fn get_first_user_key(
+    reader: &mut MvccReader<impl Snapshot>,
+    start_key: &Key,
+    end_key: &Key,
+) -> Option<Key> {
+    // The start key from TiDB is actually a range, which is used to limit the
+    // upper bound of this flashback when scanning data, so is not a real key.
+    let (keys, _) = reader
+        .scan_latest_user_keys(Some(start_key), Some(end_key), |_, _| true, 1)
+        .unwrap();
```

Comment:
Make `get_first_user_key` return `TxnResult<Option<Key>>` and use `?` here. Do not unwrap directly.